app/models/cache/Cache.scala

```diff
@@ -26,7 +28,24 @@ case class Cache(providerId: String, answers: Option[Answers], queryUcr: Option[
 }
 
 object Cache {
-  implicit val format: OFormat[Cache] = Json.format[Cache]
+  private val stringDateReads: Reads[Instant] = implicitly[Reads[String]].map(Instant.parse)
+
+  private val mongoDateReads: Reads[Instant] = MongoJavatimeFormats.instantReads
+
+  private val instantReads: Reads[Instant] = stringDateReads orElse mongoDateReads
```

Comment:
Should this not try to read as a mongo date first (the correct format) and then only try string format if mongo format fails? :thinking:

Reply:
It will work as is but all new records will be mongo format so probably worth having that first